cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ShowProceduresTest.scala

```diff
@@ -28,60 +28,158 @@ class ShowProceduresTest extends DocumentingTest {
+        p.columnAs[List[String]]("rolesExecution").foreach(_ should be(null))
+        p.columnAs[List[String]]("rolesBoostedExecution").foreach(_ should be(null))
+      })) {
+        limitedResultTable(List("name", "description", "rolesExecution", "rolesBoostedExecution"))
       }
-      section("Listing procedures with other filtering") {
-        database("system")
-        initQueries(
-          "CREATE USER jake SET PASSWORD 'abc123' CHANGE NOT REQUIRED"
-        )
-        database("neo4j")
-        p(
-          """The listed procedures can also be filtered by whether a user can execute them.
-            #This filtering is only available through the `EXECUTABLE` clause and not through the `WHERE` clause.
-            #This is due to using the users privileges instead of just filtering on the available output columns.""".stripMargin('#'))
-        p(
-          """There are two versions of the `EXECUTABLE` clause.
-            #The first one is to filter for the current user:""".stripMargin('#'))
-        login("jake", "abc123")
-        query("SHOW PROCEDURES EXECUTABLE BY CURRENT USER YIELD *", ResultAssertions(p => {
-          p.columns should contain theSameElementsAs Array("name", "description", "mode", "worksOnSystem", "signature", "argumentDescription", "returnDescription", "admin", "rolesExecution", "rolesBoostedExecution", "option")
-          p.columnAs[List[String]]("rolesExecution").foreach(_ should be(null))
-          p.columnAs[List[String]]("rolesBoostedExecution").foreach(_ should be(null))
-        })) {
-          limitedResultTable(List("name", "description", "rolesExecution", "rolesBoostedExecution"))
-        }
-        p("Notice that the two roles columns are empty due to missing the <<administration-security-administration-dbms-privileges-role-management, SHOW ROLE>> privilege.")
-        logout()
-        p("The second version filters the list to only contain procedures executable by a specific user:")
-        query("SHOW PROCEDURES EXECUTABLE BY jake", ResultAssertions(p => {
-          p.columns should contain theSameElementsAs Array("name", "description", "mode", "worksOnSystem")
-        })) {
-          limitedResultTable(List("name", "description", "mode", "worksOnSystem"))
-        }
+      p("Notice that the two roles columns are empty due to missing the <<administration-security-administration-dbms-privileges-role-management,SHOW ROLE>> privilege.")
+      logout()
+      p("The second version filters the list to only contain procedures executable by a specific user:")
+      query("SHOW PROCEDURES EXECUTABLE BY jake", ResultAssertions(p => {
+        p.columns should contain theSameElementsAs Array("name", "description", "mode", "worksOnSystem")
+      })) {
+        limitedResultTable(List("name", "description", "mode", "worksOnSystem"))
```

Comment:
All descriptions render as code blocks and within quotation marks. It's not strictly wrong, but it doesn't look so nice.